src/mod_mam_utils.erl

```diff
@@ -969,11 +969,22 @@ maybe_previous_id(X) ->
     X - 1.
 
 
--spec is_last_page(PageSize, TotalCount, Offset, MessageRows) -> boolean() when
+%% @doc Returns true, if there are more pages to paginate in the result set.
```

Comment:
This is why I disagree with over-commenting - the function is called `is_complete_result` which suggests that it returns `true` if this is the final page of the result. The comment says something opposite though. The function name is right while the comment is already outdated on introduction.

Reply:
yep, should be "if there are no more pages"...

Reply:
fixed the comment.
Also added some more info, about future optimizations (the one, that my client uses).